src/ensembl/src/content/app/entity-viewer/gene-view/styles/_constants.scss

```diff
@@ -1,5 +1,14 @@
+$gene_image_width_value: 695;
+$gene_image_width: $gene_image_width_value * 1px;
+
 $left_column: 160px;
 $middle_column_left_gap: 18px;
-$middle_column: 695px;
+$middle_column: $gene_image_width;
 $middle_column_right_gap: 27px;
 $right_column: 180px;
+
+/* stylelint-disable */
+:export {
+  gene_image_width_value: $gene_image_width_value;
```

Comment:
wouldn't you prefer to export it under the name gene_image_width (without `value`)?

Reply:
OK